plugins/pangea/textGuard.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  HookEventType,
+  PluginContext,
+  PluginHandler,
+  PluginParameters,
+} from '../types';
+import { post, getText } from '../utils';
+import { VERSION } from './version';
+
+export const handler: PluginHandler = async (
+  context: PluginContext,
+  parameters: PluginParameters,
+  eventType: HookEventType,
+  options: {}
+) => {
+  let error = null;
+  let verdict = false;
+  let data = null;
+  try {
+    const text = getText(context, eventType);
```

Comment:
This only works with `completions` request, which in this case could fail the request and can be confusing to the user. I believe pangea does support `chatCompletions` guard i.e `prompt-guard` as well. Maybe can you integrate that as well or return early if text is falsy.

Reply:
@b4s36t4 thanks a lot for your review. I've addressed your other comments but I'm missing this one. I'm not sure if I follow you, could you explain it a little bit more what would you like I implement and why it wouldn't work? You mean not all request has the field `text` set? What other fields should I check? 
I've just added the empty text error.

Reply:
Thanks for the update, I think this should work. No updates needed.